### PR TITLE
google-genai[patch]: update google/generative-ai version

### PR DESCRIPTION
**Pull request by HenryHengZJ**
Fixes [#5070](https://github.com/langchain-ai/langchainjs/issues/5070)
Allow `gemini-1.5-pro-latest` to be usable using latest google/generative-ai version

### CODE REVIEW

File: libs/langchain-google-genai/src/chat_models.ts [hunk 1]
```diff
@@ -186,7 +186,7 @@ export class ChatGoogleGenerativeAI
   private client: GenerativeModel;
 
   get _isMultimodalModel() {
-    return this.model.includes("vision");
+    return this.model.includes("vision") || this.model === 'gemini-1.5-pro-latest';
```
Comment (afirstenberg) [inline libs/langchain-google-genai/src/chat_models.ts:189]: I think locking it to gemini-1.5-pro-latest will cause frustration later.
Perhaps a leading string check on "gemini-1.5"?

**Comment by afirstenberg** [comment]
See also #5070 
Although the package makes the apiVersion "v1beta", it would be good to take this opportunity to add it for the model attribute.

**Comment by afirstenberg** [comment]
I think the changes here are pretty modest, and certainly good to get the library that supports Gemini 1.5 in is useful.
This is a prerequisite for all of the other new features (at least in the google-genai package).

File: libs/langchain-google-genai/src/chat_models.ts (hunk 1 above)
Comment (HenryHengZJ) [reply libs/langchain-google-genai/src/chat_models.ts:189]: thats a good suggestion, we can do that

**Comment by HenryHengZJ** [comment]
> See also #5070 Although the package makes the apiVersion "v1beta", it would be good to take this opportunity to add it for the model attribute.

what do you mean by the model attribute?

**Comment by afirstenberg** [comment]
> > See also #5070 Although the package makes the apiVersion "v1beta", it would be good to take this opportunity to add it for the model attribute.
> 
> what do you mean by the model attribute?

Yeah, that was a bit confusing on my part. Sorry.

What I meant to say is that it would be a good idea to add it as a attribute anywhere that "modelName" is also an attribute, with a way to set it anywhere that "modelName" can be set.

**Comment by afirstenberg** [comment]
I would have still liked to see support for an `apiVersion` parameter, but at this point I think it should just be released ASAP. (@jacoblee93, @bracesproul)

**Comment by jacoblee93** [comment]
Thank you!

**Pull request merged**